usaspending_api/urls.py

```diff
@@ -36,6 +36,7 @@
     url(r'^api/v2/financial_balances/', include('usaspending_api.accounts.urls_financial_balances')),
     url(r'^api/v1/tas/', include('usaspending_api.accounts.urls_tas')),
     url(r'^api/v1/references/', include('usaspending_api.references.urls')),
+    url(r'^api/v2/references/', include('usaspending_api.references.urls_v2')),
```

Comment:
I altered this in my branch a bit, but let's group all the v2 endpoints together below all the v1 endpoints so it's consolidated as much as possilbe